CHANGES.md

```diff
@@ -81,6 +81,7 @@
 
 ## Deprecations
 
+* Support for flink scala 2.12, because most of the libraries support version 2.12 onwards. ([beam-14386](https://issues.apache.org/jira/browse/BEAM-14386))
```

Comment:
Should this go to breaking changes?

Reply:
I already moved it to the Breaking section

Reply:
there was no change from your last commit?